### PR TITLE
update yt-channel-info

### PR DESCRIPTION
**Pull request by ChunkyProgrammer**
---
update yt-channel-info
---

**Important note**
We may remove your pull request if you do not use this provided PR template correctly.

**Pull Request Type**
Please select what type of pull request this is:
- [ ] Bugfix
- [x] Feature Implementation

**Related issue**
pr #1568 

**Description**
Please write a clear and concise description of what the pull request does.
This includes fixes to the channel community parsing as well as adds the ability to parse Channel Stats, channel links & partial support for channel home pages,

**Testing (for code that is not small enough to be easily understandable)**
Has this pull request been tested?
Tested searching & navigating to all tabs on a channel's page. Haven't tested fetch subscription without RSS nor data import/export but they should be working.

**Desktop (please complete the following information):**
 - OS: Windows
 - OS Version: 10
 - FreeTube version: 0.16.0

**Additional context**
Might want a yt-channel-info patch release before this gets merged

### CODE REVIEW

File: src/renderer/components/data-settings/data-settings.js [hunk 1]
```diff
@@ -1303,7 +1303,7 @@ export default Vue.extend({
 
     getChannelInfoLocal: function (channelId) {
       return new Promise((resolve, reject) => {
-        ytch.getChannelInfo(channelId, 'latest').then(async (response) => {
```
Comment (ChunkyProgrammer) [inline src/renderer/components/data-settings/data-settings.js:1306]: not sure why 'latest' was provided here?

**Comment by ChunkyProgrammer** [comment]
relies on https://github.com/FreeTubeApp/yt-channel-info/pull/61 getting merged

File: src/renderer/components/data-settings/data-settings.js (hunk 1 above)
Comment (PrestonN) [reply src/renderer/components/data-settings/data-settings.js:1306]: Looks like a leftover from an early revision. Doesn't seem like it's needed.

**Comment by PrestonN** [comment]
LGTM

**Pull request merged**